Build fails everywhere except Linux, Windows, macOS

```
$ git clone https://github.com/DataDog/dd-trace-go  
$ cd dd-trace-go
$ GOOS=freebsd GOOARCH=amd64 go build ./ddtrace/tracer
# [...]
# gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer
ddtrace/tracer/osinfo_default.go:19:9: undefined: unknownVersion
```

--- a/ddtrace/tracer/osinfo_default.go
+++ b/ddtrace/tracer/osinfo_default.go
@@ -16,5 +16,5 @@ func osName() string {
 }
 
 func osVersion() string {
-	return unknownVersion
+	return unknown
 }

